### ISSUE TITLE
Vendor collections to make isolated jobs work in Ansible 2.10

### ISSUE THREAD

**Issue by AlanCoding** [ansible/awx#6930]
##### ISSUE TYPE
 - Feature Idea

##### SUMMARY
Isolated jobs are made possible by a few [internal playbooks](https://github.com/ansible/awx/tree/devel/awx/playbooks) used to start, check, heartbeat, and cleanup. This is also true for container groups. In Ansible 2.10 those playbooks will no longer work because content used by them has been moved out of ansible-base.

We recently began to vendor inventory plugins in https://github.com/ansible/awx/pull/6283, the same pattern can be used for isolated playbooks. As far as I know, this will require:

 - `community.kubernetes` for `kubectl` connection plugin
 - `ansible.posix.synchronize` for `synchronize` module

So we need to put those in a folder at build time, and then change the isolated logic to set the collections path for their location.

**Comment by AlanCoding** [comment]
We apply various Ansible settings here for management playbooks

https://github.com/ansible/awx/blob/eafb751ecc4b73d65feb44218be01b2be409f14f/awx/main/isolated/manager.py#L74

so it doesn't seem crazy at all that we just add collections paths to this

**Comment by AlanCoding** [comment]
This is done now, @kdelee just want to put it on your radar, verifying isn't urgent.

Also, this loses support for Ansible 2.7, haven't tested 2.8. This could be mitigated after the tombstoning PR in Ansible core is merged, but I don't want to assume it's a problem until we have a specific reason to believe it is.

**Comment by kdelee** [comment]
@AlanCoding but we are still good w/ ansible 2.9 -- right?

**Comment by AlanCoding** [comment]
yes

I don't anticipate any other changes, unless this moves into an exec environment, which is impossible to answer right now.

**Comment by kdelee** [comment]
we could hackily verify this but as far as I'm concerned to verify it nicely we are blocked by https://github.com/ansible/tower-packaging/issues/661

**Comment by kdelee** [comment]
Moved this to 4.0.0 because we have no plans to make ansible 2.10+ work as system ansible for 3.8.0

**Comment by kdelee** [comment]
this is superseded by plans to use receptor to control remote execution and for all ansible execution to occur inside of containers, which themselves would deal with ansible content dependencies (execution environementst).
